modules/cli/src/main/scala/scala/cli/commands/new/NewOptions.scala

```diff
@@ -0,0 +1,25 @@
+package scala.cli.commands.`new`
+
+import caseapp.*
+import scala.cli.commands.shared.*
+
+@HelpMessage(NewOptions.newMessage, "", NewOptions.detailedNewMessage)
+final case class NewOptions(
+  @Recurse
+  shared: SharedOptions = SharedOptions()
+) extends HasSharedOptions
```

Comment:
Do I really need `HasSharedOptions` here? I kinda added it to just satisfy `ScalaCommand`